mne/decoding/transformer.py

```diff
@@ -17,7 +18,10 @@
 
 
 class Scaler(TransformerMixin):
-    """Standardize data across channels.
+    u"""Standardize data across channels.
+
+    By default, this makes each time point (within each epoch) have
```

Comment:
I think this is a bug. The aim of the scaler is originally to make the EEG/grad and mag varying around the same scale. It should remove the channel mean and divide by its deviation over time samples (i.e. all time points => meaning time * epochs). 

This can change significantly some results e.g. `make_pipeline(Scaler(), Xdawn())`